src/reports/AppointmentResourcePlanning/RenderAppointmentResourcePlanningReport.tsx

```diff
@@ -40,12 +40,12 @@ function RenderAppointmentResourcePlanningReport(props: RenderAppointmentResourc
     } = props;
 
     const reportExportHeaders = [
-        { label: "Questionnaire", key: "questionnaire_name" },
```

Comment:
All these style changes make this hard to review. Would be much better to have not had them in this PR.

Reply:
Doh! That's the auto-formatting, that was never meant to be in there :(